pkg/sql/crdb_internal.go

```diff
@@ -1372,13 +1371,18 @@ CREATE TABLE crdb_internal.builtin_functions (
 	},
 }
 
+// Prepare the row populate function.
+var typeView = tree.NewDString("view")
+var typeTable = tree.NewDString("table")
+var typeSequence = tree.NewDString("sequence")
+
 // crdbInternalCreateStmtsTable exposes the CREATE TABLE/CREATE VIEW
 // statements.
 //
 // TODO(tbg): prefix with kv_.
-var crdbInternalCreateStmtsTable = virtualSchemaTable{
-	comment: `CREATE and ALTER statements for all tables accessible by current user in current database (KV scan)`,
-	schema: `
+var crdbInternalCreateStmtsTable = makeAllRelationsPGCatalogTable(
```

Comment:
Maybe this should be renamed to something like `makeAllRelationsVirtualTableWithIndex` or something, since its definitely going to see use outside of the pg_catalog